### DISCUSSION TITLE
Let's Brainstorm!

### DISCUSSION THREAD

**Discussion by Akshu-on-github** [Akshu-on-github/MLHINIT2022#2] [Ideas]
Hey there everyone!

I'm so glad that we have this chance to collaborate together! Since communication is one of the toughest challenges of remote collaboration, I created this discussion to facilitate the same 😃

**Comment by prathimacode-hub** [comment]
Yeah, right. Happy to collaborate. So let's plan it out. Have you done any of the challenges for the day? @Akshu-on-github

**Reply by Akshu-on-github** [reply]
Yup - I've been working on the `Create a 3D model`, `Build with a Cloud Technology`, `Create a Tutorial for Something you Learned this Week` and the `Build an Environmental Hack/Support the Environment` challenges. If I get them done soon, I'm planning to work on the `Build a Water Hack` challenge!

**Reply by prathimacode-hub** [reply]
We shall do one thing, we can plan on topics we would work on. So, we can collaborate in other projects based on what I or you had completed from the challenges. But even I don't have access to Twitter API. If we couldn't get, we can add up as separate task. I will do up a task of sentiment analysis, password manager and social media API.

**Reply by prathimacode-hub** [reply]
You give me your email address, I shall add you up once I am done. Can you add me up for 3D Model and Building with Cloud Technology after you are done working? Currently I am working on password manager. @Akshu-on-github

**Reply by Akshu-on-github** [reply]
That sounds good! I'll DM you my email on Discord

**Reply by Akshu-on-github** [reply]
Lemme rename this repo to Day 5 MLH Init 2022 then - we can add our projects onto this!

**Reply by Akshu-on-github** [reply]
Penny for your thoughts @Ankit0225?

**Reply by prathimacode-hub** [reply]
Sure, I did create my repo with same title. @Akshu-on-github

**Comment by Akshu-on-github** [comment]
Here are a few resources that I think captures the essence of sentiment analysis:

<details>
<summary>A Beginner’s Guide to Sentiment Analysis with Python <a href="https://towardsdatascience.com/a-beginners-guide-to-sentiment-analysis-in-python-95e354ea84f6">(Link)</a></summary>
Things I found interesting
<ul>
<li>Generating wordclouds
<li>Positive/negative scores
</ul>
</details>

<details>
<summary>Python Sentiment Analysis Tutorial <a href="https://www.datacamp.com/community/tutorials/simplifying-sentiment-analysis-python">(Link)</a></summary>
Things I found interesting:
<ul>
<li>Bag of words
<li>Naive Bayes classifier
<li>A list of next steps at the end
</ul>
</details>

I'll add more as I find them - please feel free to do the same!

**Reply by Akshu-on-github** [reply]
@prathimacode-hub and @Ankit0225 - do either of you have access to the Twitter API? If so, we could try applying the sentiment analysis to tweets. My application is still under review 😓

**Reply by Akshu-on-github** [reply]
We could also use an Online Reviews API, but we can cross-submit it to the `Use a Social Media API` if we use the Twitter API

**Comment by prathimacode-hub** [comment]
Yes, thanks for the resources. I'll send by if i get any. No, I don't have access to Twitter API. I'll see, if i could ask someone. @Akshu-on-github

**Reply by prathimacode-hub** [reply]
That's a wonderful thought. It would clear "Social Media API" "Sentiment Analysis" "Also Submit Challenge with Someone in Guild". It would be a triple bonanza. Wow. @Akshu-on-github

**Comment by Akshu-on-github** [comment]
Hey everyone,
Please don't be alarmed - I've removed collaborator access for each of you, but you can still contribute via PRs! This is because I realized that this repo would be a good place to organize all of my INIT submissions
